mw-webapp/src/dictionary/CommonContent.json

```diff
@@ -23,6 +23,12 @@
     "ua": "Політика конфіденційності",
     "ka": "Კონფიდენციალურობის პოლიტიკა"
   },
+  "termsAndConditions": {
+    "ru": "Условия и положения",
```

Comment:
We don;t have this right now? let's don't create text that we can't use)

Reply:
No, we don't have it. Ok :( I thought it would be needed in the future